bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
```

Comment:
How about using shell versions of bools rather than magic strings?

run_ci = 0

if [[ ${run_ci} -eq 1 ]]
then
  ....

Reply:
done